pulser-core/pulser/channels/base_channel.py

```diff
@@ -78,6 +78,7 @@ class Channel(ABC):
     min_avg_amp: int = 0
     mod_bandwidth: Optional[float] = None  # MHz
     eom_config: Optional[BaseEOM] = field(init=False, default=None)
+    _has_fixed_addressing: bool = field(init=False, default=False)
```

Comment:
So I agree that it looks a lot like a ClassVar, I will make it so. 

This is to prevent the generation of DMM by `DMM.Global` and `DMM.Local`. I thought we would like this for all the child classes of Channel having only one type of addressing (that's why I named it has_fixed_adressing)

Reply:
In fact there is another solution: I should use typing.get_args to get the argument of addressing.